test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -6,4 +6,14 @@ struct IntBox {
   IntBox operator+(IntBox rhs) { return IntBox{.value = value + rhs.value}; }
```

Comment:
It might be also a good idea to change `operator+` into `operator-` to ensure that arguments are passed in the correct order.

Reply:
Good idea. It looks like we're doing it correctly (yay). See #33866.